doc-content/enterprise-only/springboot/openshift-springboot-proc.adoc

```diff
@@ -0,0 +1,168 @@
+. If necessary, you configured Spring security for the application. For instructions about configuring Spring security, see xref:bus-app-security-con_{context}[].
+. You are logged on to your {OPENSHIFT} environment using the `oc` command and the required project is active.
+ifdef::PAM[]
+. If you want to use {CENTRAL} Monitoring, you installed {CENTRAL} Monitoring using the operator. For instructions about installing {CENTRAL} Monitoring and other {PRODUCT} components using the operator, see {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_].
++
+[IMPORTANT]
+====
+You must configure {CENTRAL} Monitoring to use the controller startup strategy. To enable the controller strategy on {CENTRAL} Monitoring, in the *Console* tab of the operator configuration, add a `KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED` environment variable and set it to `false`.
+====
+endif::PAM[]
+
+.Procedure
+
+. In the `business-application-kjar` project directory, create a `business-application-service.xml` file with the following content:
++
+[source,xml]
+----
+<kie-server-state>
+ <controllers/>
+ <configuration/>
+ <containers>
+   <container>
+     <containerId>@project.name@_@project.version@</containerId>
+     <releaseId>
+       <groupId>@project.groupId@</groupId>
+       <artifactId>@project.name@</artifactId>
+       <version>@project.version@</version>
+     </releaseId>
+     <status>STARTED</status>
+     <scanner>
+       <status>STOPPED</status>
+     </scanner>
+     <configItems>
+       <config-item>
+         <name>KBase</name>
+         <value></value>
+         <type>BPM</type>
+       </config-item>
+       <config-item>
+         <name>KSession</name>
+         <value></value>
+         <type>BPM</type>
+       </config-item>
+       <config-item>
+         <name>MergeMode</name>
+         <value>MERGE_COLLECTIONS</value>
+         <type>BPM</type>
+       </config-item>
+       <config-item>
+         <name>RuntimeStrategy</name>
+         <value>PER_PROCESS_INSTANCE</value>
+         <type>BPM</type>
+       </config-item>
+     </configItems>
+     <messages/>
+     <containerAlias>@project.name@</containerAlias>
+   </container>
+ </containers>
+</kie-server-state>
+----
++
+. In the `business-application-kjar` project directory, edit the `pom.xml` file. Under the `<build>` tag, add the following lines:
++
+[source,xml]
+----
+  <resources>
+    <resource>
+      <directory>${basedir}</directory>
+      <includes>
+        <include>business-application-service.xml</include>
+      </includes>
+      <filtering>true</filtering>
+    </resource>
+  </resources>
+----
++
+. Outside the project directories, create an `ocp-image` directory with the following subdirectories:
++
+----
+ocp-image
+|--/root
+  |--/opt
+    |-- /m2
+      |--  /repo
+    |-- /spring-service
+----
++
+. In the `root/opt/m2` subdirectory, create a `settings.xml` file with the following content:
```

Comment:
typo In the root `/opt/m2`

Reply:
This whole XML approach is imho obsolete and we are not recommending it to a customer anymore.

Consider documenting this approach instead:
https://issues.redhat.com/browse/BAPL-1531

Reply:
I agree with @agiertli This BAPL makes the whole procedure obsolete.
It is a really complex BAPL which requires poring over.
@ramendik maybe this paragraph should be discarded, I found that another doc jira was created:
https://issues.redhat.com/browse/BXMSDOC-6414